tools/xcms_macro/repository_dependencies.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0"?>
 <repositories>
     <repository name="no_unzip_datatype" owner="lecorguille" />
```

Comment:
Maybe this could be removed as well in the future if we would have something like https://github.com/galaxyproject/galaxy/pull/13568#issuecomment-1525877683 .. but, not entirely sure.

Reply:
I think your right, this is an old option made before the use of DataCollection. @lecorguille What do you think of the idea?

Reply:
Many Galaxy instances already have disabled TS-datatypes and the new toolshed will drop them entirely.
I think this file should be removed.